### ISSUE TITLE
Adapt to Google Play Games

### ISSUE THREAD

**Issue by kauron** [silva96/guessthehashtag#4]
The records are kept in an online database, but Google Play Games has already an implementation for that (not implying that it's easier, but that is part of the Android environment)

**Comment by silva96** [comment]
Yes I know, could be a nice feature! right now the only thing we are storing is a max streak, max score and week score.
